tests/test_search_pso.c

```
@@ -0,0 +1,51 @@
+#include "search.h"
+
+// cppcheck-suppress missingIncludeSystem
+#include <math.h>
+// cppcheck-suppress missingIncludeSystem
+#include <stdio.h>
+
+#include "test_support.h"
+
+static real quadratic_2d(real *x)
```

Clang (14.0.6)

[warning] parameter 'x' is unused

(misc-unused-parameters)